src/wp-admin/includes/ajax-actions.php

```diff
@@ -5326,13 +5330,15 @@ function wp_ajax_health_check_dotorg_communication() {
  * @see WP_REST_Site_Health_Controller::test_background_updates()
  */
 function wp_ajax_health_check_background_updates() {
+	_deprecated_function( __FUNCTION__, '5.6.0', 'WP_REST_Site_Health_Controller::test_background_updates()' );
+
 	_doing_it_wrong(
 		'wp_ajax_health_check_background_updates',
 		sprintf(
 		// translators: 1: The Site Health action that is no longer used by core. 2: The new function that replaces it.
 			__( 'The Site Health check for %1$s has been replaced with %2$s.' ),
 			'wp_ajax_health_check_background_updates',
-			'WP_REST_Site_Health_Controller::test_background_updates'
+			'WP_REST_Site_Health_Controller::test_background_updates()'
 		),
 		'5.6.0'
 	);
```

Comment:
Should the `_doing_it_wrong()` be removed now there will be a `_deprecated_function()` call ? The information provided is essentially the same, so this seems redundant.